ie_serving/server/start.py

```diff
@@ -72,10 +72,12 @@ def serve(models, max_workers: int=1, port: int=9000):
         sys.exit(0)
 
 
-def start_web_rest_server(models, rest_port):
+def start_web_rest_server(models, rest_port, num_threads):
     d = PathInfoDispatcher({'/': create_rest_api(models)})
     server = WSGIServer(('0.0.0.0', rest_port), d,
-                        numthreads=1, request_queue_size=50)
+                        numthreads=num_threads,

```

Comment:
Please check which parameter in that WSGIServer exactly mean how many request can be handle in one time.

Reply:
Looking at:
https://docs.cherrypy.org/projects/cheroot/en/latest/pkg/cheroot.wsgi.html#cheroot.wsgi.Server

numthreads - "Set minimum number of threads."
For maximum number of threads, by default, there's no limit.
Can't find any other parameter related to parallelism or multiple server threads.